src/main/java/org/elmlang/intellijplugin/Elm.bnf

```diff
@@ -87,8 +87,10 @@ exposing_clause ::= EXPOSING exposed_values { methods = [isExposingAll getRefere
 comment ::=
     LINE_COMMENT
     | level_comment
+    | doc_comment
 
 level_comment ::= START_COMMENT COMMENT_CONTENT* END_COMMENT
+doc_comment ::= START_DOC_COMMENT COMMENT_CONTENT* END_DOC_COMMENT
```

Comment:
`doc_comment ::= START_DOC_COMMENT COMMENT_CONTENT* END_COMMENT` is a better idea. `END_DOC_COMMENT` would only be an alias for `END_COMMENT` which I find misleading.